src/toxcore/dht/server/mod.rs

```diff
@@ -635,9 +635,11 @@ impl Server {
     fn send_nat_ping_req_inner(&self, friend: &DhtFriend, nat_ping_req_packet: DhtRequest)
         -> impl Future<Item = (), Error = TimeoutError<mpsc::SendError<(Packet, SocketAddr)>>> + Send {
         let packet = Packet::DhtRequest(nat_ping_req_packet);
-        let futures = friend.close_nodes.nodes.iter().map(|node| {
-            self.send_to_node(node, &packet)
-        }).collect::<Vec<_>>();
+        let futures = friend.close_nodes.nodes
+            .iter()
+            .flat_map(|node| node.to_packed_node().into_iter())
```

Comment:
How does it fix the issue?

Reply:
`to_packed_node` takes only one address from two and returns `Option`.

Reply:
> takes only one address from two

based on IPv4/v6?

Reply:
Based on last response.

Reply:
OK, thanks